crates/cargo-wdk/src/trace.rs

```diff
@@ -57,3 +57,32 @@ pub fn get_cargo_verbose_flags<'a>(
         _ => Some("-vv"),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use clap_verbosity_flag::Verbosity;
+
+    #[test]
+    fn map_input_clap_verbosity_flags_to_cargo_flags() {
+        // (incoming verbosity, expected cargo flag Option)
+        let cases = vec![
```

Comment:
What happens if someone instantiated `Verbosity` with illegal values e.g. `15` for the first argument or `3` for the second argument?

Do we want to add tests for some of these values?

Reply:
The first argument represents the count of verbose flags, and the second indicates the count of quiet flags. An [internal offset](https://github.com/clap-rs/clap-verbosity-flag/blob/v3.0.2/src/lib.rs#L208) is calculated as `first - second`. When this value exceeds 3, it sets the trace level. In our case, we have one such scenario that covers that i.e `(4, 0)`.

Reply:
Okay, can you add another one which does not have a `0` in it because `0` is an edge condition as well?